src/ontology/cl-edit.owl

```diff
@@ -19367,14 +19367,15 @@ SubClassOf(Annotation(oboInOwl:is_inferred "true") obo:CL_0002572 obo:CL_0000134
 
 # Class: obo:CL_0002573 (Schwann cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm") obo:IAO_0000115 obo:CL_0002573 "A glial cell that ensheathes axons of neuron in the peripheral nervous system and are necessary for their maintainance and function.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm") obo:IAO_0000115 obo:CL_0002573 "A glial cell that ensheathes an axon in the peripheral nervous system, which is necessary for the maintenance and function of the ensheathed neurons.")
```

Comment:
After a cursory review, it seems that this definition is incomplete:

`One of the most important functions of the Schwann cell is to myelinate the axons of the PNS. Myelin, which is a fatty layer that insulates the axon, helps to increase the saltatory conduction of the neuron. A myelinating Schwann cell wraps around a single axon. Myelination occurs in larger diameter axons (≳1 μm), while a process called ensheathment occurs on smaller diameter axons in which a single cell wraps around multiple small, unmyelinated axons separating them with a thin layer of cytoplasm.` [PMID:16807057](https://doi.org/10.1016/j.biocel.2006.05.007)

`Axons regulate the Schwann cell phenotype, determining whether they myelinate individual axons or ensheathe multiple, small axons in Remak bundles.`  [PMID:23279426](https://pubmed.ncbi.nlm.nih.gov/23279426)

The current definition seems to be focused on non-myelinating Schwann cells.

Reply:
Agreed. Text def updated, comment added and dbxref added.